### DISCUSSION TITLE
Does it has smth like renderOption prop?

### DISCUSSION THREAD

**Discussion by dannicsitnikov** [mantinedev/mantine#5463] [Q&A]
### Dependencies check up

- [X] I have verified that I use latest version of all @mantine/* packages

### What version of @mantine/* packages do you have in package.json?

6.0.19

### What package has an issue?

@mantine/core

### What framework do you use?

Next.js

### In which browsers you can reproduce the issue?

None

### Describe the bug

Hey there! I've a task to render custom option label at Select component. Does it has smth like renderOption prop? For example - I need to translate option label. How I can do it? I didn't find smth which could help me

### Self-service

- [X] I would be willing to implement a fix for this issue

**Comment by rtivital** [comment]
https://v6.mantine.dev/core/select/#custom-item-component

**Reply by dannicsitnikov** [reply]
@rtivital ah, thanks a lot!

**Reply by dannicsitnikov** [reply]
It's strange but TS says `Component definition is missing display name`:

<img width="508" alt="Screenshot 2023-12-22 at 13 07 13" src="https://github.com/mantinedev/mantine/assets/33753366/495a3715-a658-4d16-a6b4-d1070fd7a707">

**Reply by Toshinaki** [reply]
@dannicsitnikov  https://stackoverflow.com/questions/52992932/component-definition-is-missing-display-name-react-display-name
